cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -977,4 +989,24 @@ export default {
     cy.do(viewHoldingsButtonByID(holdingsID).click());
     cy.expect(Button('Actions').exists());
   },
+
+  checkMarcAppIconExist: (indexRow) => {
+    cy.expect(
+      contributorAccordion
+        .find(MultiColumnList({ id: 'list-contributors' }))
+        .find(MultiColumnListRow({ index: indexRow }))
+        .find(marcAuthorityAppIcon)
+        .exists(),
+    );
+  },
+
+  checkMarcAppIconAbsent: (indexRow) => {
+    cy.expect(
+      contributorAccordion
+        .find(MultiColumnList({ id: 'list-contributors' }))
```

Comment:
Please create a const for 'MultiColumnList({ id: 'list-contributors' })' interactor as it is used multiple times

Reply:
resolved